applications/nrf_desktop/doc/dfu.rst

```diff
@@ -85,6 +85,7 @@ fwinfo
 
    * Version and length of the image.
    * Partition ID of the image, used to specify the image placement on the flash.
+   * Bootloader variant
```

Comment:
From device perspective bootloader variant is a separate option. It's not part of the fwinfo. I already mentioned this in my previous comment

Reply:
Yes it is I added it to it
When you run fwinfo it will show you also bootloader variant

Reply:
However it is not in class fwinfo

Reply:
It's description from perspective of an application module. Discussed offline.